validator/packaging/ubuntu/postinst

```diff
@@ -25,7 +25,8 @@ directories="
 user="sawtooth"
 group="sawtooth"
 
-if [ "$1" = install ]; then
+# Run on install and not upgrade
+if [ "$1" = configure ] && [ -z "$2" ]; then
```

Comment:
What is the expected value of `$2`?  It's not clear from looking at this code.

Reply:
-z returns True if the string length is zero. When dpkg does an new install it calls 'postinst configure' but on an upgrade it calls 'postinst configure previousversion'